src/test/groovy/com/ableton/JenkinsMocksTest.groovy

```diff
@@ -6,13 +6,21 @@ import static org.junit.Assert.assertTrue
 
 import com.lesfurets.jenkins.unit.BasePipelineTest
 import org.junit.After
+import org.junit.Before
 import org.junit.Test
 
 
 /**
  * Tests for the JenkinsMocks class.
  */
 class JenkinsMocksTest extends BasePipelineTest {
+  @SuppressWarnings('UnnecessaryOverridingMethod')
```

Comment:
I agree with CodeNarc, why are we overriding this method?

Reply:
For reasons that I don't entirely understand, without this override the `currentBuild` variable is not correctly bound and the tests fail with this error:

```
Caused by: groovy.lang.MissingPropertyException: No such property: currentBuild for class: groovy.lang.Binding
	at groovy.lang.Binding.getVariable(Binding.java:58)
	at groovy.lang.Binding$getVariable$0.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
```

Reply:
Could you paste a link to the failing Jenkins job? I would like to see the whole thing in context.

Reply:
@ala-ableton I only ever saw the error locally. However I just pushed a WIP commit with should also fail on Jenkins (hopefully 😅 ).

Reply:
@ala-ableton Ok, it just failed on Jenkins as well: http://newci.office.ableton.com:8080/blue/organizations/jenkins/AbletonDevTools-jenkins-pipeline-mocks/detail/AbletonDevTools-jenkins-pipeline-mocks/40/tests